src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json

```diff
@@ -3610,7 +3730,57 @@
       }
     }
   },
-  "slider-handle-size-extra-large": {
+  "slider-handle-height-precision-small": {
+    "value": "24px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-handle-height-precision-small"
+      }
+    }
+  },
+  "slider-handle-height-precision-medium": {
+    "value": "26px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-handle-height-precision-medium"
+      }
+    }
+  },
+  "slider-handle-height-precision-large": {
+    "value": "30px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-handle-height-precision-large"
+      }
+    }
+  },
+  "slider-handle-height-precision-extra-large": {
+    "value": "32px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-handle-height-precision-extra-large"
+      }
+    }
+  },
+  "slider-handle-precision-width": {
+    "value": "6px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-handle-precision-width"
+      }
+    }
+  },
+  "🚫 slider-handle-size-extra-large": {
```

Comment:
Commented in [our Slack thread](https://adobedesign.slack.com/archives/GTZ7NSNR0/p1722275123818209?thread_ts=1722040533.723329&cid=GTZ7NSNR0). Tokens marked for deprecation, which will point to modified versions of these token names, should update their values to the new tokens that will be used instead.